Sources/JWTKit/JWK.swift

```diff
@@ -5,6 +5,36 @@
 public struct JWKS: Decodable {
     /// All JSON Web Keys
     public var keys: [JWK]
+
+    /// Retrieves the desired key from the JSON Web Key Set
+    /// - Parameters:
+    ///   - keyIdentifier: The `kid` value to lookup.
+    ///   - keyType: The `kty` value.
+    public func find(keyIdentifier: String, keyType: JWK.KeyType) -> JWK? {
+        keys.filter { $0.keyType == keyType && $0.keyIdentifier?.string == keyIdentifier }.first
+    }
+
+    /// Retrieves the desired key from the JSON Web Key Set
+    /// - Parameters:
+    ///   - keyIdentifier: The `kid` value to lookup.
+    ///   - keyType: The `kty` value.
+    public func find(keyIdentifier: JWKIdentifier, keyType: JWK.KeyType) -> JWK? {
+        find(keyIdentifier: keyIdentifier.string, keyType: keyType)
```

Comment:
Vapor style requires `self.` everywhere.

Reply:
But...But...Ray requires *not* using self.  LOL.